core/lib/dal/src/models/storage_block.rs

```diff
@@ -79,16 +79,18 @@ impl From<StorageL1BatchHeader> for L1BatchHeader {
             bloom: H2048::from_slice(&l1_batch.bloom),
             used_contract_hashes: serde_json::from_value(l1_batch.used_contract_hashes)
                 .expect("invalid value for used_contract_hashes in the DB"),
-            base_fee_per_gas: l1_batch
-                .base_fee_per_gas
-                .to_u64()
-                .expect("base_fee_per_gas should fit in u64"),
+            base_fee_per_gas: U256::from(
+                l1_batch
+                    .base_fee_per_gas
+                    .to_u64()
```

Comment:
You should avoid casting to u64 here

Reply:
Addressed here [37fccd5](https://github.com/matter-labs/zksync-era/pull/1324/commits/37fccd5b0468b99de0d861dbde9c23f5a43bb293)